<launch>
  <node pkg="move_arm" type="teleop_arm_base" output="screen">    

    <remap from="collision_map" to="collision_map_occ" />
    <remap from="collision_map_update" to="collision_map_occ_update" />
    
    <remap from="arm_ik" to="pr2_ik_right_arm/ik_service" />	
    
    <remap from="controller_start"  to="/r_arm_joint_waypoint_controller/TrajectoryStart" />
    <remap from="controller_query"  to="/r_arm_joint_waypoint_controller/TrajectoryQuery" />
    <remap from="controller_cancel" to="/r_arm_joint_waypoint_controller/TrajectoryCancel" />

    <param name="group" type="string" value="right_arm" />

    <param name="use_planning" type="bool" value="true" />
    <param name="refresh_interval_collision_map" type="double" value="20.0" />
    <param name="refresh_interval_kinematic_state" type="double" value="1.0" />
    <param name="bounding_planes" type="string" value="0 0 1 -0.01" />
    <param name="arm_control_interval" type="double" value="2.0" />
  </node>

</launch>
